src/main/java/teamrtg/rtg/core/RTG.java

```diff
@@ -97,7 +86,6 @@ private void registerStructures()
         MapGenStructureIO.registerStructure(StructureOceanMonumentRTG.StartMonument.class, "rtg_MapGenOceanMonumentRTG");
```

Comment:
With the new ENABLE_SCATTERED_FEATURE_MODIFICATIONS config option, this would probably be appropriate:

```java
---   	// Scattered features
---      MapGenStructureIO.registerStructure(MapGenScatteredFeatureRTG.Start.class, "rtg_MapGenScatteredFeatureRTG");
---
---      // Villages
---      if (Mods.RTG.config.ENABLE_VILLAGE_MODIFICATIONS.get())
---          MapGenStructureIO.registerStructure(MapGenVillageRTG.Start.class, "rtg_MapGenVillageRTG");
---
---      // Ocean monuments
---      MapGenStructureIO.registerStructure(StructureOceanMonumentRTG.StartMonument.class, "rtg_MapGenOceanMonumentRTG");
```
```java
+++   	// Scattered features
+++      if (Mods.RTG.config.ENABLE_SCATTERED_FEATURE_MODIFICATIONS.get())
+++          MapGenStructureIO.registerStructure(MapGenScatteredFeatureRTG.Start.class, "rtg_MapGenScatteredFeatureRTG");
+++
+++      // Villages
+++      if (Mods.RTG.config.ENABLE_VILLAGE_MODIFICATIONS.get())
+++          MapGenStructureIO.registerStructure(MapGenVillageRTG.Start.class, "rtg_MapGenVillageRTG");
+++
+++      // Ocean monuments
+++      if (Mods.RTG.config.GENERATE_OCEAN_MONUMENTS.get())
+++          MapGenStructureIO.registerStructure(StructureOceanMonumentRTG.StartMonument.class, "rtg_MapGenOceanMonumentRTG");
```